src/__snapshots__/App.test.js.snap

```diff
@@ -0,0 +1,121 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[` App Component Rendering  renders correctly 1`] = `
+<div
+  className="App"
+>
+  <div>
+    <nav>
+      <h3
+        className="h3"
+      >
+        Math Magicians
+      </h3>
+      <ul
+        className="ul"
+      >
+        <a
+          href="/"
+          onClick={[Function]}
+        >
+          <li>
+            Home
+          </li>
+        </a>
+        <a
+          href="/calculator"
+          onClick={[Function]}
+        >
+          <li
+            className="link"
+          >
+            Calculator
+          </li>
+        </a>
+        <a
+          href="/quote"
+          onClick={[Function]}
+        >
+          <li
+            className="link"
+          >
+            Quote
+          </li>
+        </a>
+      </ul>
+    </nav>
+  </div>
+  <div>
+    <h1>
+      Welcome to our Page
+    </h1>
+    <p>
+      Lorem ipsum odor amet, consectetuer adipiscing elit. Risus natoque finibus maximus laoreet efficitur aliquam morbi? Eleifend turpis natoque auctor aenean sapien congue dapibus integer. Consectetur consectetur cras, congue posuere fermentum porttitor. At iaculis tempor donec eu vel sodales urna magnis. At varius pellentesque finibus etiam conubia eget. Ornare finibus tempus quisque primis vivamus class id aptent auctor. Fames mus netus metus nisl velit dolor. Aliquet suspendisse nostra at viverra nec.
```

Comment:
- [ ] optional - when testing the file I came across a test result that said ``obsolete`` please fix this issue although It wont impede your progress